Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,136 @@
+## Table of contents
+
+* [Overview](#overview)
+* [Selectors](#selectors)
+* [Events on selected elements](#events-on-selected-elements)
+* [Points to note while migrating](#points-to-note-while-migrating)
+* [Example migration](#example-migration)
+
+## Overview
+
+Currently, Oppia End-to-End tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The GSoC project **Migrate away from Protractor** aims to migrate all the existing End-to-End tests to WebdriverIO.
+
+This guide is to help contributors in migrating end-to-end tests from Protractor to WebdriverIO during the hybrid state.
+
+## Selectors
+
+A selector is used to query an element. The WebDriver Protocol provides several selector strategies to query an element. WebdriverIO simplifies them to keep selecting elements simple
```

Comment:
You missed the addition of a period at the end from my [original comment](https://github.com/oppia/oppia-web-developer-docs/pull/7#discussion_r900411468):

```suggestion
A selector is used to query an element. The WebDriver Protocol provides several selector strategies to query an element. WebdriverIO simplifies them to keep selecting elements simple.
```

Reply:
Done.